### ISSUE TITLE
Cannot correct first message in interactive learning (rasa x UI)

### ISSUE THREAD

**Issue by Krogsager** [RasaHQ/rasa#4814]
**Rasa version**: 1.4.5 

**Rasa X version**: 0.2.22

**Python version**: 3.7.5

**Operating system**:  Ubuntu 18.04.3 LTS 

**Issue**: 
TL:DR The "Correct this" button does not work on the first message. But it works on later messages and responses

Via Rasa X UI, in interactive learning:
I've said "goodbye". The bot recognized *greet intent.
I want to correct the intent so I click "Correct this" in the UI.
The UI does not respond
![image](https://user-images.githubusercontent.com/27994384/69336987-13fd9100-0c60-11ea-8dea-a4803991d895.png)

**Error (including full traceback)**:
` WARNING  rasax.community.api.blueprints.stack  - Updating events for sender 'a642834e031046d7b57bfffd74a23f6a' failed. Error: 400, Bad Request, body='b'{"version":"1.4.5","status":"failure","message":"Couldn\'t extract a proper event from the request body.","reason":"BadRequest","details":{"parameter":"","in":"body"},"help":null,"code":400}''` 

**Command or request that led to error**: N/A (UI interaction)

**Comment by sara-tagger** [comment]
Thanks for raising this issue about Rasa X 🔧@gausie will get back to you about it soon!

###### In the meantime, you may find some useful information in the [Rasa X docs](https://rasa.com/docs/rasa-x/) or by searching our [forum](https://forum.rasa.com/c/rasa-x) 😊

**Comment by DiegoProtec** [comment]
I don't know if the same occurs as here: [copy to interactive](https://github.com/RasaHQ/rasa/issues/4788), is same error?
